src/app/core/common-components/basic-autocomplete/basic-autocomplete.component.ts

```diff
@@ -209,6 +209,11 @@ export class BasicAutocompleteComponent<O, V = O>
     });
   }
 
+  getPlainTextTooltip(htmlString: string): string {
+    // Remove HTML tags for tooltip display
+    return htmlString.replace(/<[^>]*>/g, "");
```

Comment:
let's not use html in strings and then parse it out again at other places. Why do we need this here? Let's better add explicit css classes based on conditions to the html templates to have clear control